File: Display_image_and_video/videocapture.cpp
#include <iostream>
#include <opencv2/opencv.hpp>
using namespace std;
using namespace cv;

int main(int argc,char **argv)
{

//	VideoCapture video(0); //o for webcam
//	VideoCapture video(1); //1 for USB camera device.
	VideoCapture video(argv[1]); //Input from the command line (Any stored videos or IP camera address.)
	if(!video.isOpened())
		cout<<"Error ,vodeo can't be loaded"<<endl;

	double fps=video.get(CV_CAP_PROP_FPS);
	cout<<"FPS"<<fps<<endl;
	Mat img; 

	for(;;)
	{
		//Mat img(640,480, CV_8UC1, Scalar(70));  creating the empty model image
		
		bool success=video.read(img);
		if(!success)
			break;

		imshow("video",img);
		if(waitKey(3)==27)

			break;
	}




}

File: Display_image_and_video/videocapture.py
#videocapture.py
import cv2
import sys

#open the video into the capture object...........
#capture = cv2.videocapture(sys.argv[1]) #input from command line
capture = cv2.VideoCapture(0)
width = int(capture.get(3))
height = int(capture.get(4))

while(True):
	#Reading the continuous frames
	_,frame = capture.read()

	if frame.any():
		print("reading the frame from videos....")
	else:
		print("Error,while reading the frames from camera...........")

	cv2.namedWindow("video_frames",cv2.WINDOW_NORMAL)
	cv2.imshow("video_frames",frame)
	cv2.waitKey(2)
